indico/modules/rb_new/client/js/common/bookings/BookingEdit.jsx

```diff
@@ -200,10 +206,13 @@ class BookingEdit extends React.Component {
                     isFetching: false,
                     data: {
                         ...currentBooking.data,
+                        pendingCancellations: {},
                         candidates,
                         conflicts,
                         conflictingCandidates,
-                        pendingCancellations: {},
+                        blockings,
+                        unbookableHours,
+                        unbookablePeriods,
```

Comment:
Other bookings are still missing here.

Reply:
Yeha, I know. I am still thinking how to properly fix.

Reply:
Ah, sure, I thought this was ready.

Reply:
No problem. I should have created a Draft PR ;)

Reply:
But unfortunately, once you create a PR you can't make convert it to a draft.

Reply:
Yeah, it's a pity. You could always list WIP items in PR description